src/qml/NightstandPage.qml

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright (C) 2022 - Ed Beroset <github.com/beroset>
+ * Copyright (C) 2020 - Darrel Griët <idanlcontact@gmail.com>
+ * Copyright (C) 2015 - Florent Revest <revestflo@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import QtQuick 2.15
+import QtQuick.Layouts 1.3
+import org.asteroid.controls 1.0
+import org.asteroid.utils 1.0
+import org.asteroid.settings 1.0
+import org.nemomobile.systemsettings 1.0
+import Nemo.Configuration 1.0
+
+Item {
+
```

Comment:
nit: we usually try to avoid these empty lines

Reply:
fixed